packages/Babylonian-Printbugger.package/BPSwimlane.class/instance/newTitleMorphFor..st

```diff
@@ -1,14 +1,19 @@
 private
 newTitleMorphFor: aBPExample
 	
-	^ Morph new
+	| exampleLabel |
+	exampleLabel := ((aBPExample uniqueIdentifier subStrings: '>>') fold:[:aString :otherString| aString, Character cr, otherString]) asText.
+	exampleLabel addAttribute: (BPPluggableTextAttribute new evalBlock: [
+		ToolSet 
+			browse: aBPExample method methodClass 
+			selector: aBPExample method selector asSymbol]).
+	^ (TextMorph string: exampleLabel size: TextStyle defaultFont height)
```

Comment:
Same comments as for packages/BabylonianPrintbugger.package/BPSwimlane.class/instance/newLineNameMorphFor..st

Reply:
In my opinion it's also more of an overhead to transform it into a button than simply making use of a Text's Attribute, because

    when passing the title as a string for the label, it'll be transformed into a StringMorph that does not show linebreaks (String cr).
    when transforming the title into 3 separate morphs and adding them to the button morph the (default) label is still visible and taking up additional space.

Also I tried using a simple Morph to which I added the title. A Morph offers addMouseUpActionWith: codeToRun to add actions. However, using that message also adds fancy mouse-enter effects etc. that you can't get rid of without additional overhead.

Reply:
New Solution is now using `Morph on:send:to:` for the on-click action